config/taxon_blacklist.txt

```diff
@@ -0,0 +1,597 @@
+-1
```

Comment:
is -1 always blacklisted?  what if that's something genuinely never seen before? :P)

Reply:
i think right now -1 goes under genus -200 rather than -201 but anyway we will need to reaggregate

Reply:
```
mysql> select * from taxon_lineages where taxid < 0;
Empty set (0.00 sec)
```
So this means -1 will always be put under -200, might as well exclude it from -201.